### ISSUE TITLE
In the settings can you add preferred download quality

### ISSUE THREAD

**Issue by Cliffard** [LagradOst/CloudStream-3#780]

**Comment by Zeroo097** [comment]
Tap and hold on the episode or movies you want to download and a bunch of options will appear just tap on download mirror and choose your preferred download quality although it would be nice if they add an option in settings.

**Comment by Cliffard** [comment]
@Zeroo097 yeah i know but it's better if they add preferred download quality in the settings

**Comment by Zeroo097** [comment]
@Cliffard yeah that would be good.
